changelog.d/clickhouse_acknowledgements.breaking.md

```diff
@@ -0,0 +1,3 @@
+Updated the end status for end-to-end acknowledgments for the clickhouse sink.
+
+When requests encounter 500-level errors, the end status is now "Errored" instead of "Rejected".
```

Comment:
```suggestion
Requests that encounter 500-level errors from the Clickhouse server will be retried instead of immediately producing an error at their source.
```